docs/faq.md

```diff
@@ -10,7 +10,7 @@ This FAQ is a work in progress designed to answer the questions our users most f
 
 #### **Q: Can I open Cluster Manager in Rancher Desktop?**
 
-**A:** No, the Cluster Manager feature is currently a Rancher-only concept. We are working on an early integration of the Rancher Dashboard in an upcoming release and will notify the community when this becomes available.  
+**A:** Yes, the Rancher Dashboard is included as a feature preview in the release 1.2.1. Invoke the dashboard by clicking on **Dashboard** option in the system tray menu. 
```

Comment:
I think the question needs to be updated to say `Cluster Explorer`. The `Cluster Manager` is a different piece, and the Dashboard is normally referred to as `Cluster Explorer`. The `Cluster Manager` is just for managing clusters, and is not available with Rancher Desktop.